content/README.md

```diff
@@ -0,0 +1,87 @@
+# Common Workflow Language Website 2.0
+
+A Rewrite and expansion of the [CWL Website](https://commonwl.org), using Jekyll. The current staging site is https://github.com/lunacodes/cwl-staging
+
+## Staging Issues
+
+* Top Nav needs to re-integrate logic checks for http and https
+
+## Video Player
+
+* Subtitles/Captions needed
+* Move progress bar above controls?
+* Change background color on Play, to avoid whitewash
+
+## Future Tasks & Ideas
+
+* [ ] Custom Video Player & Twitter Feeds
+* [ ] Make Search field functional
+* [ ] Split website into multiple pages
+* [x] Create video player for embedded vids
+  * [ ] Subtitles
+  * [x] Full screen button
+* [ ] Explanatory button(?)
+* [ ] Donations sub-page
+* [x] Setup staging site
+	* [ ] Setup non-GitHub Pages staging site
+* [ ] GDPR - Waiting for content from Sarah for this
+* [ ] Add Footer Styles
+
+### Twitter
+
+* Twitter's official embed offers [Opt-out of tailoring Twitter](https://developer.twitter.com/en/docs/twitter-for-websites/privacy). Is this sufficient, privacy-wise?
+  * Check what data it actually collects
+
+### Content Needed
+
+* Explanatory Button(?)
+* Page Divisions
+* Donations Page?
+* GDPR if relevant
+
+#### GDPR Resources
+
+* https://www.solodev.com/blog/web-design/gdpr-and-informed-consent-cookies-bar.stml
+* https://www.webtoffee.com/sample-css-code-snippets-for-customizing-pre-defined-templates/
+* https://cookie-script.com/
+
+## Staging Site
+
+* Currently setup at [lunacodes.github.io/cwl-staging](https://lunacodes.github.io/cwl-staging/). Need to migrate off of GitHub Pages.
+
+## Deployment
+
+* Setup Google Analytics
+* Double-check Aria Roles and other accessibility
+* Minor: Revert Table of Contents nav links to old style
+  * Old style: `#Repositories_of_CWL_Tools_and_Workflows`
+  * New style: `#repos`
```

Comment:
Can we have both?

Reply:
What's the status here?

Reply:
I'll revert them to the old version.

Reply:
Actually - as I'm looking through it, the reason I had changed them was for easy of use (i.e. it's easy to type or remember `#repos` instead of `#Repositories_of_CWL_Tools_and_Workflows`. There's also a scrolling issue with using the old format, that I'd have to resolve.

Can we leave them in the new format?

Reply:
HTML allows adjacent anchors. I like the new style, but I don't want to break existing links. So that's why I asked for both styles.

Reply:
Ok - I'll look at it after the weekend

Reply:
Fixed now, with commits 92c9e74, df94c96, 2ff139f